### ISSUE TITLE
Status bar HEIGHT PROPERTY doesn't work when ZENMODE is ON

### ISSUE THREAD

**Issue by darianmorat** [drcika/apc-extension#157]
The option for height in the status bar is no longer working:
```json
"apc.statusBar": {
      "height": 26,
   }
```
I've already restarted VSCode, removed the extension, installing everything again, but is not working. By the way, this is the only option/setting that is not working, the rest seems just fine. Thanks

**Comment by Arastais** [comment]
I am having the same issue, but with the `activityBar` property. It broke after updating VSCodium to `1.87.0.24060`:
```json
"apc.activityBar": {
    "position": "bottom",
    "hideSettings": true,
}
```

**Comment by drcika** [comment]
![Screenshot 2024-03-04 at 05 38 29](https://github.com/drcika/apc-extension/assets/35919606/8e257d31-e8b3-4f0c-b83e-f9bbd69bc1dc)


everything seems to be fine

**Comment by darianmorat** [comment]
I did realize something, actually the property for height of the status bar is buggy when Zen Mode is used... 
Steps to reproduce the bug:

- Use ZenMode with status bar visible: `"zenMode.hideStatusBar": false` 
- Set an specific height for status bar with apc: `"apc.statusBar": {"height": 30}`
- Now, this is not gonna work, so the workaround is first to set up the height on apc and after that, toggle Zen Mode
- Now close VSCode and open it again, the option is going to be reset and u need to apply the changes again in order to have the custom height

**Comment by Arastais** [comment]
> I did realize something, actually the property for height of the status bar is buggy when Zen Mode is used...

Personally, I wasn't using zen mode originally, and changing my zen mode setting had no effect. My activity bar is still stuck on the right no matter what apc settings are changed. Even the settings icon is still present with `"hideSettings": true,` active.

**Comment by thisisxjj** [comment]
> I am having the same issue, but with the `activityBar` property. It broke after updating VSCodium to `1.87.0.24060`:
> 
> ```json
> "apc.activityBar": {
>     "position": "bottom",
>     "hideSettings": true,
> }
> ```

Yes I have the same issue. I try to uninstall VsCode and remove all extensions. then install vscode and all extensions, remove all APC extension settings, rewrite just below setting, Now all work fine.
![image](https://github.com/drcika/apc-extension/assets/46477356/7510bd8c-5cad-45b2-8289-72e2eb05903f)
![image](https://github.com/drcika/apc-extension/assets/46477356/7fa7bcbc-ddaf-4478-96ee-a818f76f4df4)
